src/Parts/Guild.php

```diff
@@ -33,15 +33,15 @@ class Guild
     public MessageNotificationLevels $default_message_notifications;
     public ExplicitContentFilterLevels $explicit_content_filter;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Role[]
+     * @var Role[]
      */
     public array $roles;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Emoji[]
+     * @var Emoji[]
      */
     public array $emojis;
     /**
-     * @var \Ragnarok\Fenrir\Enums\Parts\GuildFeatures[]
+     * @var GuildFeatures[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)